plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/exec/DBCStatistics.java

```diff
@@ -151,25 +151,27 @@ public boolean isEmpty() {
     }
 
     public void accumulate(DBCStatistics stat) {

```

Comment:
Consider adding `@Nullable` annotation since this method now contains `null` checks.